.github/workflows/option.yml

```
@@ -0,0 +1,2 @@
+client:
```

actionlint (1.7.12)

[error] "jobs" section is missing in workflow

(syntax-check)